.gitignore

```diff
@@ -3,6 +3,7 @@
 /deps
 erl_crash.dump
 *.ez
+mix.lock
```

Comment:
We don't wanna ignore mix.lock :)

In the Ruby world it's common and I used to do it, but I argued with José once and he had (of course!) very valid points, e.g. a common experience for maintainers etc. - so we'd wanna keep this around :)